tests/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8084",
+  "projectId": "ufkudc"
```

Comment:
Will need to be replaced if dataverse were to use Cypress (and dashboard) at some point